imod/mf6/out/__init__.py

```diff
@@ -69,6 +69,10 @@ def open_cbc(
     "disu": disu.open_cbc,
 }
 
+_OPEN_DVS = {
+    "dis": dis.open_dvs,
```

Comment:
Can we also add disv support?

Reply:
Just did, but I don’t seem to find a disv testmodel including an uzf-package?

Reply:
That makes sense.

In principle, we should be to add automatically generated DISV models from our DIS test models, since xugrid provides `from_structured` to generate equivalent UgridDataArrays from ordinary DataArrays.

This seems like a good reason to priortize the `from_dataset` for Datasets in xugrid: https://github.com/Deltares/xugrid/issues/8

Then, we could add a `to_disv` method on our packages which returns the Dataset, with a UgridDataset in lieu of the xr.Dataset. This is also a good check in general whether identical models in DIS vs DISV form return the same results.

But it's a separate issue. Can you add a `TODO: ` here for now?

Reply:
I've created #1246

Reply:
I just added a test for a DISV model with UZF-package. Needed some minor fixes in the UZF-package as well. Wat I also found is that .open_cbc(flowja=False) crashes on the model budgets. Something goes wrong in the disv_to_horizontal_lower_indices(). The method works is flowja=True. It has maybe something to do with inactive cells. I will make a separate issue for this.

Reply:
I've merged #1250 which should fix inactive DISV cells.